### PR TITLE
Adding Cyclus Installation Instructions

### PR DESCRIPTION
**Pull request by gtw2**
## Summary of changes

Adding a Cyclus installation guide to the `guides` page.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] I have read the **CONTRIBUTING** document.
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

## Associated Issues and PRs

- Issue: #

## Associated Developers

- Dev: @

### CODE REVIEW

File: manual/guides/cyclus.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+Make sure your Ubuntu environment is clean to ensure the script works.
+
+## Step 1. Get the Script
+Download the script from [here](https://github.com/arfc/scripts/setmyenv). Make sure you have
+both the `startup.sh` and `toBash.txt` files.
+
+## Step 2. Run the script
+Change to the directory containing the script and proceed to run it with the following:
+```
+$ bash startup.sh
```
Comment (katyhuff) [inline manual/guides/cyclus.md]:
```suggestion
cd setmyenv && bash startup.sh
```

File: manual/guides/cyclus.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+|Install Cyclus Dependencies (apt) | `2` |
+|Install Conda 3 | `1` |
+|Install Cyclus Dependencies (conda) | `1` |
+|Install Pyne | `1` |
+|Install Jupyter Extensions | `1` (Optional) |
+|Setup Git | `1` |
+|Install Sublime Text | `1` (Optional) |
+|Install Other Software | `2` |
+
+** `1` and `2` Correspond to the displayed Yes and No options respectively.
```
Comment (katyhuff) [inline manual/guides/cyclus.md]:
```suggestion
** `1` and `2` correspond to the displayed Yes and No options respectively.
```

File: manual/guides/cyclus.md [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+
+## Step 3. Clone Cyclus and Cycamore Repositories
+The repositories are found here for [cyclus](https://github.com/cyclus/cyclus) and [cycamore](https://github.com/cyclus/cycamore).
```
Comment (katyhuff) [inline manual/guides/cyclus.md]:
```suggestion
The repositories are found here for [cyclus](https://github.com/cyclus/cyclus) and 
[cycamore](https://github.com/cyclus/cycamore).
```

File: manual/guides/cyclus.md [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+
+Likewise, the following can be entered in the terminal:
+```
+$ cd
```
Comment (katyhuff) [inline manual/guides/cyclus.md]:
```suggestion
cd ~ && git clone https://github.com/cyclus/cyclus && git clone https://github.com/cyclus/cycamore
```

File: manual/guides/cyclus.md [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+$ git clone https://github.com/cyclus/cyclus
```
Comment (katyhuff) [inline manual/guides/cyclus.md]:
```suggestion
```

File: manual/guides/cyclus.md [hunk 6]
```diff
@@ -0,0 +1,71 @@
...
+$ cd
```
Comment (katyhuff) [inline manual/guides/cyclus.md]:
```suggestion
```

File: manual/guides/cyclus.md [hunk 7]
```diff
@@ -0,0 +1,71 @@
...
+$ git clone https://github.com/cyclus/cycamore
```
Comment (katyhuff) [inline manual/guides/cyclus.md]:
```suggestion
```

File: manual/guides/cyclus.md [hunk 8]
```diff
@@ -0,0 +1,71 @@
...
+```
+
+## Step 4. Installation
+With the depencies installed, and the repositories downloaded we can proceed to build from source.
+This is accomplished by running the `setup.py` scripts in each directory.
+
+```
+$ cd
```
Comment (katyhuff) [inline manual/guides/cyclus.md]:
```suggestion
cd ~/cyclus && python setup.py
```

File: manual/guides/cyclus.md [hunk 9]
```diff
@@ -0,0 +1,71 @@
...
+$ cd cyclus
```
Comment (katyhuff) [inline manual/guides/cyclus.md]:
```suggestion
```

File: manual/guides/cyclus.md [hunk 10]
```diff
@@ -0,0 +1,71 @@
...
+$ python setup.py
```
Comment (katyhuff) [inline manual/guides/cyclus.md]:
```suggestion
```

File: manual/guides/cyclus.md [hunk 11]
```diff
@@ -0,0 +1,71 @@
...
+```
+
+After Cyclus is installed, we proceed to Cycamore:
+```
+$ cd
```
Comment (katyhuff) [inline manual/guides/cyclus.md]:
```suggestion
cd ~/cycamore && python setup.py
```

File: manual/guides/cyclus.md [hunk 12]
```diff
@@ -0,0 +1,71 @@
...
+$ cd cycamore
```
Comment (katyhuff) [inline manual/guides/cyclus.md]:
```suggestion
```

File: manual/guides/cyclus.md [hunk 13]
```diff
@@ -0,0 +1,71 @@
...
+$ python setup.py
```
Comment (katyhuff) [inline manual/guides/cyclus.md]:
```suggestion
```

File: manual/guides/cyclus.md [hunk 14]
```diff
@@ -0,0 +1,71 @@
...
+```
+
+If both have successfully installed, we must now verify they perform as intended with unit tests:
+```
+$ cyclus_unit_tests
```
Comment (katyhuff) [inline manual/guides/cyclus.md]:
```suggestion
cyclus_unit_tests && cycamore_unit_tests
```

File: manual/guides/cyclus.md [hunk 15]
```diff
@@ -0,0 +1,71 @@
+---
+layout: manual
+title: Cyclus
+subtitle: "Cyclus - Creating your Ubuntu Environment"
+permalink: /manual/guides/cyclus
+---
+
+## Step 0. Clean Install Ubuntu
+Make sure your Ubuntu environment is clean to ensure the script works.
```
Comment (katyhuff) [inline manual/guides/cyclus.md]: Be clear about what you mean here.... (how would someone check for a clean environment?) 
(e.g. delete any old cyclus installations, check bashrc, etc.)

**Comment by gtw2** [comment]
I believe I addressed all of your review comments and I am waiting on @gwenchee to review or somebody to merge for me.

File: manual/guides/cyclus.md [hunk 16]
```diff
@@ -0,0 +1,67 @@
...
+title: Cyclus
+subtitle: "Cyclus - Creating your Ubuntu Environment"
+permalink: /manual/guides/cyclus
+---
+
+## Step 0. Clean Install Ubuntu
+Make sure your Ubuntu environment is clean to ensure the script works. 
+This is most easily done by performing a fresh install of Ubuntu. Otherwise,
+check that previous installations of cyclus have been removed, and the 
+bashrc file is in the default state.
```
Comment (gwenchee) [inline manual/guides/cyclus.md]: Should there be a guide or link to instructions to how to clean install ubuntu?

File: manual/guides/cyclus.md [hunk 17]
```diff
@@ -0,0 +1,67 @@
...
+The repositories are found here for [cyclus](https://github.com/cyclus/cyclus) and 
+[cycamore](https://github.com/cyclus/cycamore).
+
+Likewise, the following can be entered in the terminal:
+```
+cd ~ && git clone https://github.com/cyclus/cyclus && git clone https://github.com/cyclus/cycamore
+```
+
+## Step 4. Installation
+With the depencies installed, and the repositories downloaded we can proceed to build from source.
```
Comment (gwenchee) [inline manual/guides/cyclus.md]:
```suggestion
With the dependencies installed, and the repositories downloaded we can proceed to build from source.
```

File: manual/guides/cyclus.md (hunk 16 above)
Comment (gtw2) [reply manual/guides/cyclus.md]: We could hyperlink to [here](https://tutorials.ubuntu.com/tutorial/tutorial-install-ubuntu-desktop#0)

**Comment by robfairh** [comment]
This is very useful. I wish there was a @gtw2 in my past.

**Pull request merged**